.gitignore

```diff
@@ -1,3 +1,4 @@
 *.py[co]
 *~
 *.orig
+docs/_build/
```

Comment:
Do we still need this line in the .gitignore file?

Reply:
Do we not _not_ need it?